### PR TITLE
Add position param to imgproc shortcode

### PR DESCRIPTION
**Pull request by aormsby**
There we go! A proper branch to merge!

I noticed you had a position parameter in both the image and figure shortcodes. I also wanted to use it in the imgproc shortcode, so I added the same data in. A very small addition, quite helpful.

### CODE REVIEW

File: layouts/shortcodes/imgproc.html [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 {{ $image := .Scratch.Get "image" }}
 
-<figure>
+<figure class="{{ with .Get 3}}{{ . }}{{ else -}} left {{- end }}" >
```
Comment (panr) [inline layouts/shortcodes/imgproc.html]:
```suggestion
<figure class="{{ with .Get "position"}}{{ . }}{{ else -}} left {{- end }}" >
```

Comment (panr) [reply layouts/shortcodes/imgproc.html]: I'd prefer to stick to the named params ;-)

Comment (aormsby) [reply layouts/shortcodes/imgproc.html]: Ah, that's where things get a little tricky. We can't mix positional params with named params (throws an error), and the shortcode already uses positional params for the other data. However, it looks like we could use `.Scratch` to set our own local variable for it. I'm just unclear on how it works. I'll look into it, though.

Comment (panr) [reply layouts/shortcodes/imgproc.html]: Right, I forgot the context and didn't look at the whole implementation ;-( If your changes work, I'm fine with local "named" param ;-D

**Comment by aormsby** [comment]
Sorry, this conversation layout is a little confusing. Just to be clear, I did update the code with the local variable and committed for you. My thing about .Scratch may have been a bit off - I'm not sure I understand how that's used quite yet. However, it's working fine now. 🙂

**Comment by panr** [comment]
Code looks fine, but I think we should also update README. Add some words about this `position` param and an example that users don't have to dive into Hugo docs ;-) 

https://github.com/panr/hugo-theme-hello-friend#built-in-shortcodes

**Comment by aormsby** [comment]
Yep, made a quick addition. Hopefully it's clear enough for readers. Should be ready to go!

While I was working on this I noticed a few inconsistencies between the different image shortcodes. In imgproc there is no `alt` param for example. However, since we aren't currently using named params, I think this is fine to avoid. I started another topic on this (#133). I'll add these notes for future work.

**Comment by panr** [comment]
OMG, totally forgot about this PR 🤦 Sorry about that! I'm merging it 🎉

**Pull request merged**